File: templates/matlab/matrix_tensor_phase_portrait.m
function fig = matrix_tensor_phase_portrait()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('phase_plane', 4411, 'matrix and tensor visualization: phase portrait', 'matrix and tensor visualization', 'phase portrait');
end

File: templates/python/matrix_tensor_phase_portrait.py
"""matrix_tensor_phase_portrait: 矩阵与张量可视化相平面画像（phase-plane 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='matrix and tensor visualization: phase portrait'):
    return make_template_figure('phase_plane', seed=4411, title=title, domain='matrix and tensor visualization', topic='phase portrait')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
